modules/host/contractmanager/contractmanager.go

```diff
@@ -165,6 +165,13 @@ func newContractManager(dependencies dependencies, persistDir string) (*Contract
 		defer cm.wal.mu.Unlock()
 
 		for _, sf := range cm.storageFolders {
+			// No storage folder to close if the folder is not available.
+			if atomic.LoadUint64(&sf.atomicAvailable) == 1 {
```

Comment:
it's a bit confusing to me that `sf.atomicAvailable == 1` means the folder is unavailable and not `sf.atomicAvailable == 0`